src/pages/index.jsx

```diff
@@ -1,6 +1,7 @@
 import Link from "next/link";
 import { useTranslation } from "next-i18next";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import * as React from "react";
```

Comment:
- It is forbidden to import all from a module ❗.

Reply:
- Absolute paths should be avoided and used relative imports instead.

Reply:
this is the index ,, not my component,, I did not change anything in it.